src/pages/collections/CollectionsShowPage.js

```diff
@@ -269,6 +275,22 @@ class CollectionsShowPage extends Component {
 
                 <div className="details-section-content-grid">
                   {this.subCollectionDescription()}
+                  <div style={{ marginTop: "30px" }}>
+                    <SocialButtons
+                      buttons={[
+                        "Email",
+                        "Facebook",
+                        "Twitter",
+                        "Pinterest",
+                        "Tumblr",
+                        "Reddit"
+                      ]}
+                      url={window.location.href}
+                      image={this.state.thumbnail_path}
+                      title={this.getTitle()}
+                    />
```

Comment:
Please create a components for this so both `collectionshowpage` and `archivepage` can use it.

Reply:
I'm sorry but I don't understand. The SocialButtons.js file is a component and it has been imported on both the archive and collection pages. Does that not show up in my changes?

Reply:
https://github.com/VTUL/dlp-access/pull/246/files#diff-c550e3639b774a9f35e9d86b555322296bd4b33ecb47778126d0a340780cb1feR293 and https://github.com/VTUL/dlp-access/pull/246/files#diff-df7761cb3b73e8f53e4c2c5fa7abd224f93027ccda65f821f19da238e5a5e901R278 can be merged and to be used by both pages.

Reply:
I still am not 100% sure, but are you suggesting to merge the citation component and the social media buttons?

Reply:
1. create a component like
```
class CustomSocialButton extends Component {
    
    constructor(props) {
        super(props);
        this.state = {
            buttons: []
        };
    }

    async getButtonList() {
        let buttonlist = ["Email", "Facebook", "Twitter", "Pinterest", "Tumblr", "Reddit"];
        this.setState({ buttons: buttonlist });
    }

    componentDidMount() {
        this.getButtonList();
    }

    render() {
        return  ( 
            <SocialButtons
                buttons={this.state.buttons}
                url={window.location.href}
                {...this.props}
            />
        );
    }

}
```
2. Then both collectionshowpage and archivepage can use it, like 
archivepage
```
<CustomSocialButton
  image={this.state.item.thumbnail_path}
  title={this.state.item.title}
/>
```
and collectionshowpage
```
<CustomSocialButton
  image={this.state.thumbnail_path}
  title={this.getTitle()}
/>
```

also create a JIRA ticket to implement getButtonList() that fetch configuration options